### DISCUSSION TITLE
krita AI's region selection real-time redraw error

### DISCUSSION THREAD

**Discussion by Tasvthe** [Acly/krita-ai-diffusion#678] [General]
How to solve krita AI's region selection real-time redraw error, but select the whole layer redraw fine, use the selection tool to select a part of the layer error.
![{624B9A2C-C8B8-4e7a-8B90-3D01FC8DBCDC}](https://github.com/Acly/krita-ai-diffusion/assets/156665109/aa25fd96-d0b5-44c0-8267-6aaae779dfd8)

**Comment by Tasvthe** [comment]
![{228A315B-1D37-43e2-9351-0F27A2A01885}](https://github.com/Acly/krita-ai-diffusion/assets/156665109/408ad7c0-5132-4e3d-8c69-42f830a2e675)

**Comment by Acly** [comment]
Likely your ComfyUI version is too old, please update

**Comment by Tasvthe** [comment]
Fixed, the COMFUI version is not updated
